aim/sdk/run.py

```diff
@@ -100,6 +108,7 @@ def _close(self) -> None:
             return
         self.finalize_system_tracker()
         self.finalize_run()
+        self.finalize_remote_tracking_queue()
```

Comment:
this will do the trick for now, but I guess it's not the best idea for the run to wait until the tracking queue is fully empty, as there might be other runs in progress as well (in the same queue). Maybe put a `Todo` item for furhter improvement?

Reply:
make sense. done